ref/general/JAAS-authentication.adoc

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description:
+:seo-title: JAAS authentication with Open Liberty
+:seo-description: Open Liberty relies on the Java Authentication and Authorization Service (JAAS) framework to determine a user's identity and access roles.
+:page-layout: general-reference
+:page-type: general
+= JAAS authentication with Open Liberty
+
+Open Liberty relies on the Java Authentication and Authorization Service (JAAS) framework to determine a user's identity and access roles. JAAS authentication is suited to microservice-based applications because it separates the authentication process from the rest of the application so that authentication can be managed independently. New or updated authentication processes can be plugged in to an application without otherwise modifying the application itself.
+
+The JAAS framework defines a `Subject` object to represent the source of a request to an application. The subject can be any entity that makes a request, such as a user or another microservice. The subject contains a set of principals, or details about the identity of the entity that made the request, which is used for authentication. The subject also contains sets of credentials, or security attributes that are used for authorization. By creating a subject to represent a user or service, Open Liberty provides an efficient and easily configurable means for authentication data to pass between microservices in an application.
+
+With JAAS in Open Liberty, each stage in the authentication process operates independently and can be updated or reconfigured with minimal impact on the other stages in the process. The `LoginContext` class determines the methods that are used for authentication, according to the `LoginModule` interfaces that are configured. Each `LoginModule` interface represents a particular type of authentication service. For example, one `LoginModule` interface can handle basic username and password authentication, while another  processes SSO tokens.
```

Comment:
It looks like there might be an extra space between `another` and `processes`

Reply:
thanks- fixed